config/initializers/module_handler.rb

```diff
@@ -0,0 +1,36 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2014 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'yaml'
+config = YAML.load_file('config/configuration.yml')[Rails.env].symbolize_keys
+
+unless config[:disabled_modules].nil?
+  to_disable = config[:disabled_modules]
+  OpenProject::Plugins::ModuleHandler.disable_modules(to_disable)
+end
```

Comment:
This can be shortened by using OpenProject::Configuration['disabled_modules']. Additionally, it would follow the standard OP configuration flow.

In order to have a default, we should add ```'disabled_modules' => []``` to the default hash in ```lib/open_project/configuration.rb```